packages/styles/scss/components/data-table/_vars.scss

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../theme' as *;
+
+// TODO - REMOVE? FIND ALL FINDS NO USAGES
+// /// @type Value
+// /// @access public
+// /// @group data-table
+// $data-table-heading-transform: uppercase !default;
+
+// TODO - REMOVE? FIND ALL FINDS NO USAGES
+// /// @type Value
+// /// @access public
+// /// @group data-table
+// $data-table-heading-border-bottom: 1px solid $border-interactive !default;
+
+// TODO - REMOVE? FIND ALL FINDS NO USAGES
+// /// @type Number
+// /// @access public
+// /// @group data-table
+// $data-table-row-height: 2rem !default;
+
+// TODO - REMOVE? FIND ALL FINDS NO USAGES
+// /// @type Color
+// /// @access public
+// /// @group data-table
+// $data-table-zebra-color: $layer-accent !default;
```

Comment:
I couldn't find usage of these variables in a find all in the entire monorepo. Safe to remove?

Reply:
Do itttt

If you can add it to our v11 migration file for carbon-components that'd be awesome (just flagging them as removed)

Reply:
@joshblack Sweet! Removed and updated the migration doc